scripts/puppeteer/jest.config.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
   transform: {
     '^.+\\.tsx?$': 'ts-jest',
   },
+  moduleNameMapper: {
+    '^puppeteer-core/internal/(.*)': '<rootDir>/../../node_modules/puppeteer-core/lib/cjs/puppeteer/$1',
```

Comment:
Could you briefly explain why this had to be added? :)

Reply:
It was mentioned in the previous PR that I linked. pasting again for better visibility https://github.com/puppeteer/puppeteer/issues/9535